pkg/registries/adapters/rhcc_adapter.go

```diff
@@ -56,11 +56,11 @@ func (r RHCCAdapter) FetchSpecs(imageNames []string) ([]*apb.Spec, error) {
 	specs := []*apb.Spec{}
 	for _, imageName := range imageNames {
 		req, err := http.NewRequest("GET",
-			fmt.Sprintf("%v/v2/%v/manifests/latest", r.Config.URL.String(), imageName), nil)
+			fmt.Sprintf("%v/v2/%v/manifests/%v", r.Config.URL.String(), imageName, r.Config.Tag), nil)
```

Comment:
I had to go out of my way, by modifying the template to remove a default value for tag in order to see what happens. This looks to cover it in a case where either nothing or a empty string are passed in.

https://github.com/openshift/ansible-service-broker/pull/357/commits/044e566bcfa162f7ca5ee0763d84a541a7644f93#diff-54e88cebcb04cc49b8796b2f774cb6c7R249

Once I set the default there to "" it did break, and off into a crash loop it went..
```
[2017-08-11T13:34:37.589Z] [ERROR] Error grabbing JSON body from response: json: cannot unmarshal number into Go value of type struct { History []map[string]string "json:\"history\"" }
```

I made a change to the code to use latest when Tag comes back as "", which seemed to get the broker off and rolling.